scripts/integration_test_contracts.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+# Wait until the height of the block is greater than or equal to 1.
+h="0"
+until [[ ${h} == '"1"' ]]
+do
+  h=$(fnsad query block 1 --node http://localhost:26658 --chain-id simd-testing --keyring-backend test | jq .block.header.height)
+done
+
+# send transaction to node
+cur_path=`pwd`
+
+artifacts_path="artifacts"
+
+## parameters
+CONTRACT_ADDRESS="link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8"
+FROM_ACCOUNT="link146asaycmtydq45kxc8evntqfgepagygelel00h"
+URL="http://localhost:26658"
+CODE_ID=1
+
+# This is a function that checks if the result is as expected.
+check_result() {
+    if [[ "$result" == *"failed"* ]]; then
```

Comment:
I doubt is it enough. How about checking as before https://github.com/Finschia/finschia-wasm/pull/3/commits/8375ecb610bccdf87033878732507940ae368afe?

FYI

```
check_result() {
    if [[ "$1" != "A" || "$2" != "B" ]]; then echo "err with $3"; exit 1; fi
}
```

```
check_result A B unexpected
```

and

```
check_result A B unexpected
```

works.

Reply:
Fixed as pointed out.
[fix: modify check_result](https://github.com/Finschia/finschia-wasm/pull/3/commits/771ac1b458e220246a0d58483e295e6138e9f187)

Reply:
[fix: change check_run_info to receive a raw_log argument](https://github.com/Finschia/finschia-wasm/pull/3/commits/6885273a1dba397ca90f3288a62c61944be84c2b)